e-commerce-frontend-angular-main/src/app/components/product-detail/product-detail.component.html

```diff
@@ -22,16 +22,20 @@ <h6>Rating: &#9733;&#9733;&#9733;&#9733;&#9734;
             </h6>
             <span class="fw-bold">{{product.quantity}} </span> In Stock
         </div>
-    
-        <div class="col-md-2 p-5 border-start justify-content-center">
+        
+        <div class="col-md-2 p-5 border-start justify-content-center" class="form-group">

```

Comment:
There's a second 'class' attribute on this element, messing up the element's layout.